openfisca_aotearoa/variables/rates_rebates.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+
+# Import from openfisca-core the common python objects used to code the legislation in OpenFisca
+from openfisca_core.model_api import *
+# Import the entities specifically defined for this tax and benefit system
+from openfisca_aotearoa.entities import Propertee, Person
+from numpy import clip
+
+
+# This is a 'temporary' variable used specifically for the rates rebates
+# Due to the structure not yet being clear, this may need to be moved, renamed.
+class dependants(Variable):
```

Comment:
> I believe this will always be the case for all variables, and would remove that comment.
> If you want to carry that idea of special definition, you could encode that information and make it more discoverable by renaming the variable something like `rates_rebates.dependants` or `dependants_for_rates_rebates`, or even `declared_dependants_for_rates_rebates`.

Reply:
> Comment removed, name remains. However a naming convention for these localised variables should be discussed to avoid future conflicts